rbac/management/group/view.py

```diff
@@ -1223,7 +1248,21 @@ def roles(self, request, uuid=None, principals=None):
                     group = set_system_flag_before_update(group, request.tenant, request.user)
                     remove_roles(group, role_ids, request.tenant, request.user)
 
-            return Response(status=status.HTTP_204_NO_CONTENT)
+            response = Response(status=status.HTTP_204_NO_CONTENT)
+
+            if status.is_success(response.status_code):
```

Comment:
same(condition is not necessary  )
Also, 
can you move this block into transaction block on line 1250 ?

it will roll back whole operation when log creation will need end with error.

thanks!